addon/components/mi-date.js

```diff
@@ -1,213 +1,242 @@
 import layout from "../templates/components/mi-date";
 import MobileInputComponentMixin from "../mixins/mobile-input-component";
 import {
-  isTouchDevice
+    isTouchDevice
```

Comment:
I am not sure, but it looks like your editor isn't supporting [editorconfig](https://editorconfig.org/) - please use formatting specified in out .editorconfig file